services/addSort.js

```diff
@@ -0,0 +1,14 @@
+import { sortByNumbers, sortName } from './sorting';
+
+export default function addSort(array, filter) {
+  const sortAlphabet = (elA, elB) =>
+    !filter.name.isActive || sortName(elA.name, elB.name, filter.name.isAsc);
```

Comment:
```suggestion
  const sortAlphabet = (elA, elB) =>
    filter.name.isActive && sortName(elA.name, elB.name, filter.name.isAsc)
```

Does the same but is better understandable (imo)